docs/guides/postgres/snapshot/continuous_archiving.md

```diff
@@ -126,7 +130,7 @@ type: Opaque
 
 **Archiver Storage Backend**
 
-**wal-g** supports both _S3_ and __GCS__ cloud providers.
+**wal-g** supports _S3_, _GCS_, __Azure__, and __Swift__  as cloud providers.
```

Comment:
Need similar tables that explain the secret format for GCP, Azure, Swift in details.